apps/examples/media_test/server/server_example_linux.c

```diff
@@ -59,7 +60,8 @@ struct server_context_s {
 };
 
 struct server_context_s g_sc;
-char buffer[SEND_BUFFER_SIZE];
+char s_buffer[SEND_BUFFER_SIZE];
```

Comment:
Correct coding style is g_send_buffer / g_receive_buffer.
you can use 'buf' instead of buffer.